CSS/css_completions.py

```diff
@@ -2,174 +2,391 @@
+        '<border-image-outset>', '<border-image-repeat>'
+    ],
+    'border-image-outset': ['<length>', '<number>'],
+    'border-image-repeat': ['stretch', 'repeat', 'round', 'space'],
+    'border-image-slice': ['fill', '<number>', '<percentage>'],
+    'border-image-source': ['none', '<image>'],
+    'border-image-width': ['<length>', '<percentage>', '<number>', 'auto'],
+    'border-top | border-right | border-bottom | border-left': [
+        '<border_width>', '<border_style>', '<color>'
+    ],
+    'border-top-color | border-right-color | border-bottom-color | border-left-color': ['<color>'],
+    'border-top-left-radius | border-top-right-radius | border-bottom-right-radius | border-bottom-left-radius': ['<length>', '<percentage>'],
+    'border-top-style | border-right-style | border-bottom-style | border-left-style': ['<border_style>'],
+    'border-top-width | border-right-width | border-bottom-width | border-left-width': ['<border_width>'],
+    'bottom': ['<length>', '<percentage>', 'auto'],
+    'box-decoration-break': ['slice', 'clone'],
+    'box-shadow': ['none', 'inset', '<color>'],
+    'break-after': ['<break_before_after>', '<break_inside>'],
+    'break-before': ['<break_before_after>', '<break_inside>'],
+    'break-inside': ['<break_inside>'],
+    'caption-side': ['top', 'bottom'],
+    'clear': ['none', 'left', 'right', 'both'],
+    'clip': ['rect($1)', 'auto'],
+    'color': ['<color>'],
+    'columns': ['auto'],
+    'columns-count': ['auto', '<number>'],
+    'columns-fill': ['auto', 'balance'],
+    'columns-gap': ['normal', '<length>'],
+    'columns-rule': ['<border_width>', '<border_style>', '<color>'],
+    'columns-rule-color': ['<color>'],
+    'columns-rule-style': ['<border_style>'],
+    'columns-rule-width': ['<border_width>'],
+    'columns-span': ['none'],
+    'columns-width': ['auto', '<length>'],
+    'content': [
+        'none', 'normal', '<string>', '<uri>', 'attr($1)',
+        'open-quote', 'close-quote', 'no-open-quote', 'no-close-quote'
+    ],
+    'counter-increment': ['none', '<custom_ident>', '<integer>'],
+    'counter-reset': ['none', '<custom_ident>', '<integer>'],
+    'cursor': [
+        '<uri>', 'auto', 'crosshair', 'default', 'pointer', 'move', 'e-resize',
+        'ne-resize', 'nw-resize', 'n-resize', 'se-resize', 'sw-resize',
+        's-resize', 'w-resize', 'text', 'wait', 'help', 'progress'
+    ],
+    'direction': ['ltr', 'rtl'],
+    'display': [
+        'none', 'inline', 'block', 'contents', 'list-item', 'inline-block',
+        'inline-table', 'table', 'table-cell', 'table-column',
+        'table-column-group', 'table-footer-group', 'table-header-group',
+        'table-row', 'table-row-group', 'flex', 'inline-flex', 'grid',
+        'inline-grid', 'ruby', 'ruby-base', 'ruby-text', 'ruby-base-container',
+        'ruby-text-container', 'run-in'
+    ],
+    'empty-cells': ['show', 'hide'],
+    'flex': ['none'],
```

Comment:
That is a the default flex, which is very rarely set because it's the default. 

When using the flex shorthand, It's more common to do `flex:1` or `flex: 1 3 400px`